components/eam/bld/configure

```diff
@@ -1510,9 +1511,19 @@ else {
 
     }
 
-    if (defined $opts{'use_MMF_VT'}) {
-        $nadv += 2; # add tracers for CRM t and q variance transport
-        if ($print>=2) { print "Advected variance constituent added for MMF: 2$eol"; }
+    ### advected tracers for MMF
+    if (defined $opts{'use_MMF'}) {
+        $nadv += 8;
+        if ($print>=2) { print "Advected constituents added for MMF physics package: 8$eol"; }
+
+        # Chemistry package
```

Comment:
Why are you adding the chemistry tracers here? Shouldn't this happen where the chemistry tracers are normally added? Also I'd change the comment to "advected tracers for chemistry".

Reply:
I'll remerge this back with the block above

Reply:
removed.